lib/matplotlib/backends/backend_pdf.py

```diff
@@ -1620,11 +1618,11 @@ def draw_image(self, gc, x, y, im, dx=None, dy=None, transform=None):
                              w, 0, 0, h, x, y, Op.concat_matrix,
                              imob, Op.use_xobject, Op.grestore)
         else:
-            tr1, tr2, tr3, tr4, tr5, tr6 = transform.to_values()
+            tr1, tr2, tr3, tr4, tr5, tr6 = transform.frozen().to_values()
 
             self.file.output(Op.gsave,
+                             1, 0, 0, 1, x, y, Op.concat_matrix,
                              tr1, tr2, tr3, tr4, tr5, tr6, Op.concat_matrix,
-                             w, 0, 0, h, x, y, Op.concat_matrix,
```

Comment:
so, "w" and "h" aren't needed anymore because the transform is taking care of it all?